.github/workflows/weekly-data-and-external-tool-updater.yml

```
@@ -0,0 +1,96 @@
+name: Weekly Data and External Tool Updater
+
+# https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#permissions
+permissions:
+  actions: none
+  checks: none
+  contents: none
+  deployments: none
+  id-token: none
+  issues: none
+  discussions: none
+  packages: none
+  pages: none
+  pull-requests: write
+  repository-projects: none
+  security-events: none
+  statuses: none
+
+on:
+  schedule:
+    # Run once a week (e.g., every Monday at 01:00 UTC)
+    - cron: '0 1 * * 1'
+  workflow_dispatch: # Allows manual triggering from the Actions tab
+
+jobs:
+  update-data-files:
+    runs-on: ubuntu-latest
+
+    env:
+      BUNDLE_WITHOUT: "coverage development pcap"
+
+    strategy:
+      fail-fast: true
+      matrix:
+        ruby:
+          - '3.1'
```

Small nitpick but should we use 3.2 if we can, just to future proof by one minor version?  
```suggestion
          - '3.2'
```
Seems like the majority of workflows are on 3.2 now.
```
➜  workflows git:(1462875819) ✗ grep -r "\'3\.2\'" ./ | wc -l
       6
➜  workflows git:(1462875819) ✗ grep -r "\'3\.1\'" ./ | wc -l
       4
➜  workflows git:(1462875819) ✗ grep -r "\'3\.0\'" ./ | wc -l
       1
```

---

I have no opinion on this, so if you want to make the change and land it cool, if not I'll be able to do it later today